src/test/java/Homework17.java

```diff
@@ -0,0 +1,99 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class Homework17 extends BaseTest {
+
+    @Test
+    public void addSongToPlaylist() throws InterruptedException {
+
+
+        String expectedSongAddedMessage = "Added 1 Song into \"Prepare for battle.\"";
```

Comment:
Hi,
I have checked your homework 18, you're doing great. You used the <methods> & <include> tags into Homework18 and your test script Homework18 Passed. Keep up the good work!

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/44592718/df25c1c5-9ef3-4ffe-8023-e56d79bd29d1)

But there is a small note in Homework 17, it failed to add step assertion at the end as image above. 
```
java.lang.AssertionError: 
Expected :Added 1 Song into "Prepare for battle."
Actual   :Added 1 song into "Prepare for battle."
```
Please check it again.

Best wishes from the HW team (Khanh, khanh@testpro.io)